src/ssl.c

```diff
@@ -29067,7 +29067,7 @@ int wolfSSL_BIO_vprintf(WOLFSSL_BIO* bio, const char* format, va_list args)
             break;
 
         case WOLFSSL_BIO_MEMORY:
-    #if defined(OPENSSL_EXTRA) && (!defined(_WIN32) || defined(HAVE_VA_COPY))
```

Comment:
Windows has versions of va_copy, vsnprintf, and va_end, so I don't think this needs to be gated on !_WIN32 anymore. The library built fine for me on Windows in Visual Studio. I also built the library in Cygwin and ran the unit tests and those passed.

This was the only instance of HAVE_VA_COPY in the project, so I got rid of it.